src/main/java/com/google/api/codegen/transformer/PageStreamingTransformer.java

```diff
@@ -64,6 +64,17 @@
     return descriptors;
   }
 
+  public List<PageStreamingDescriptorClassView> generateDescriptorClasses(
+      DiscoGapicInterfaceContext context) {
+    List<PageStreamingDescriptorClassView> descriptors = new ArrayList<>();
+
+    for (com.google.api.codegen.discovery.Method method : context.getPageStreamingMethods()) {
```

Comment:
Reverting the Discovery additions in this class from this PR; the paging/batching/etc transformers aren't ready yet for discogapic gen.